.maestro/setup_ui_tests.sh

```diff
@@ -0,0 +1,112 @@
+### Set up environment for UI testing
+
+## Constants
+derived_data_path=DerivedData
+app_location=$derived_data_path/Build/Products/Debug-iphonesimulator/DuckDuckGo.app
+device_uuid_path="$derived_data_path/device_uuid.txt"
+
```

Comment:
Since these should stay in sync with run_ui_tests, should we put them in a shared file and source it here and in the other script?

Reply:
Yep! 😅